### DISCUSSION TITLE
After open Rocksdb & LMQ, consume target LMQ fails

### DISCUSSION THREAD

**Discussion by 1294566108** [apache/rocketmq#7978] [Q&A] [answered]
We open rocksdb store and LMQ(No proxy open in 5.2.1 version)
but we can't consume target LMQ;
But If I close rocksdb and only open LMQ
I can consume succesfully.

**1.Enable Rocksdb & LMQ but no proxy start**
nohup sh bin/mqnamesrv &
nohup sh bin/mqbroker -n localhost:9876 -c /XX/broker.conf &

**broker.conf** file:
![image](https://github.com/apache/rocketmq/assets/91858554/be0c3727-2317-49f3-956e-2c15cffb4c62)

**2.RMQ-Dashboard show the correct config**
![image](https://github.com/apache/rocketmq/assets/91858554/854bf57d-2572-463f-9584-8001465d867f)
![image](https://github.com/apache/rocketmq/assets/91858554/a2bf65bf-ea23-477b-b80f-cf0bacc9dbf3)

**3.start producer**
![image](https://github.com/apache/rocketmq/assets/91858554/47bce8d9-acc4-4b85-9e79-ec0ecec84b81)

**4.start consumer**
![image](https://github.com/apache/rocketmq/assets/91858554/e4393211-5976-4842-b59e-3d4b681d274a)

**5.producer send successful**
![image](https://github.com/apache/rocketmq/assets/91858554/be19c710-1ed9-4ba6-a101-f63b881adb31)

**6.no consumerqueue or .sst file created in consumequeue directory AND consume fail**
![image](https://github.com/apache/rocketmq/assets/91858554/1cec3a62-49ed-4207-b3cf-6241646162eb)

**Comment by 1294566108** [comment]
@RongtongJin I can't find out why this problem occurs， I saw that sometimes sst files are generated under the consumequeue file, but they can never be consumed.

**Comment by cserwen** [comment, marked as answer]
It is currently not supported, but I think this feature is very useful. It will be a huge optimization for scenarios where MQTT supports millions of queues. We should support it. @1294566108

**Comment by RongtongJin** [comment]
https://github.com/apache/rocketmq-mqtt/issues/251#issuecomment-2026790498
